readme.md

```diff
@@ -213,6 +213,7 @@
 - [DOS](https://github.com/balintkissdev/awesome-dos#readme) - Operating system for x86-based personal computers that was popular during the 1980s and early 1990s.
 - [Nix](https://github.com/nix-community/awesome-nix#readme) - Package manager for Linux and other Unix systems that makes package management reliable and reproducible.
 - [Integration](https://github.com/stn1slv/awesome-integration#readme) - Linking together different IT systems (components) to functionally cooperate as a whole.
+- [Low-code](https://github.com/zenitysec/awesome-low-code#readme) - Allowing business professionals to address their needs on their own with little to no coding skills.
```

Comment:
> Low Code is written in multiple different ways, both with and without dash and titlized and not. It should be consistent.

Reply:
> > Low Code is written in multiple different ways, both with and without dash and titlized and not. It should be consistent.

Thank you for spotting that. Sorry, it seams like I missed a few instances. I fixed it and double checked everything.

Reply:
Not fixed

Reply:
Fixed now